listenbrainz/webserver/static/js/src/RecentListens.test.tsx

```diff
@@ -96,8 +96,9 @@ describe("Recentlistens", () => {
 
     timeago.ago = jest.fn().mockImplementation(() => "1 day ago");
     const wrapper = mount<RecentListens>(
-      <RecentListens {...props} />,
-      mountOptions
+      <GlobalAppContext.Provider value={mountOptions.context}>
```

Comment:
After hiding delete listen button when isCurrentUser is false in previous commit, I noticed the updated snapshot did not have any Delete Listen buttons. Looking at the test, I figured due to not wrapping in a context, it was causing the current user to be null. 

Fix: wrap in context because having current user in the snapshot seems more useful. If it was intentional to leave out the context here, can revert the change. 

(Snapshot still had to be updated because adding currentUser causes other changes too.)